homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,20 @@
 #include <string>
```

Comment:
Please never insert in Your commit message vulgarism or similar improper names.
Always describe what you commit contains.